package.json

```diff
@@ -15,7 +15,7 @@
     "pre-commit-lint": "npx lint-staged"
   },
   "dependencies": {
-    "@aries-framework/core": "0.1.0-alpha.276",
+    "@aries-framework/core": "^0.1.0-alpha.289",
     "@aries-framework/react-hooks": "^0.1.2",
     "@aries-framework/react-native": "0.1.0-alpha.276",
```

Comment:
React native version should stay in sync with core version

Reply:
Ah, thanks for the head up. I'll update that as well.